### PR TITLE
Drop code support for Upstart and Nagios resources and rkt bundles

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this (for merging after #3659 is merged)?

File: doc/sphinx/Pacemaker_Explained/resources.rst [hunk 1]
```diff
@@ -159,59 +158,6 @@ The ``stonith`` standard is used for managing fencing devices, discussed later
 in :ref:`fencing`.
 
 
-.. index::
```
Comment (nrwahl2) [inline doc/sphinx/Pacemaker_Explained/resources.rst:166]: Could also drop `nagios` from the `pacemakerd --features` output in Clusters from Scratch

**Review by nrwahl2 — COMMENTED** [review]
Need to update Pacemaker Explained for `rkt`, for example https://github.com/nrwahl2/pacemaker/commit/99c3b5500a12c4ba109748c291f24af4b0b1274e

File: doc/sphinx/Pacemaker_Explained/resources.rst (hunk 1 above)
Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/resources.rst:166]: That's still the correct output as long as we use AlmaLinux 9 there (though the output could use updating for the latest 9)

**Comment by kgaillot** [comment]
Updated rkt commit for doc update

**Comment by kgaillot** [comment]
Rebased on current main to resolve trivial conflict in rkt commit

File: doc/sphinx/Pacemaker_Explained/resources.rst (hunk 1 above)
Comment (nrwahl2) [reply doc/sphinx/Pacemaker_Explained/resources.rst:166]: Unless it was built without nagios support. I was just thinking of removing that as the last mention. But it's completely unimportant.

**Review by nrwahl2 — APPROVED** [review]

File: doc/sphinx/Pacemaker_Explained/resources.rst (hunk 1 above)
Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/resources.rst:166]: we use the stock AlmaLinux packages in the doc, so it's there :)

**Pull request merged**